.eslintrc

```diff
@@ -2,7 +2,8 @@
   "extends": "eslint-config-discourse",
   "rules": {
     "discourse-ember/global-ember": 2,
-    "eol-last": 2
+    "eol-last": 2,
+    "no-restricted-globals": 0
```

Comment:
What's this for? Doesn't feel related to the PR? 🤔

Reply:
It is! I'm using `Promise.allSettled()` and that function is available on native Promises, but not in RSVP. In our shared eslint config we were prohibiting the use of global (native) `Promise`, but it might be time to start transitioning off RSVP.
So for now I'm disabling it for core, and have another PR ready that replaces most RSVP uses with native promises. If that works, I'll be dropping that rule from the shared config 😃